tests/ckeditor.js

```diff
@@ -104,19 +104,6 @@ describe( 'CKEditor Component', () => {
 				expect( vm.value ).to.equal( '' );
 			} );
 
-			it( 'should set the initial data', done => {
```

Comment:
This test should not be removed. We need to be sure that the specified data has been passed to the editor.